quantized_mesh_encoder/ecef.py

```diff
@@ -16,9 +16,14 @@ def to_ecef(positions, ellipsoid=WGS84):
 
     Args:
         - positions: expected to be an ndarray with shape (-1, 3)
-        - ellisoid: a dict that defines an ellipsoid with with "a" and "b" values. Defaults to WGS84
-    from latitude-longitude-height to ecef
+        from latitude-longitude-height to ecef
+        - ellipsoid: (`Ellipsoid`): ellipsoid defined by its semi-major `a`
+          and semi-minor `b` axes. Default: WGS84 ellipsoid.
     """
+    assert isinstance(
+        ellipsoid,
+        Ellipsoid), 'ellipsoid must be an instance of the Ellipsoid class'
```

Comment:
There are some edge cases in the formatting setup I use, can you use
```suggestion
        Ellipsoid), ('ellipsoid must be an instance of the Ellipsoid class')
```
and then reformat? Should look better.

Reply:
yapf still wants to format it as below when i run `pre-commit` locally
```diff
-    assert isinstance(ellipsoid, Ellipsoid), ('ellipsoid must be an instance of the Ellipsoid class')
+    assert isinstance(
+        ellipsoid,
+        Ellipsoid), ('ellipsoid must be an instance of the Ellipsoid class')
```

Reply:
Weird. Oh well.